client/my-sites/post-type-list/post-thumbnail.jsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import resizeImageUrl from 'lib/resize-image-url';
+import { getNormalizedPost } from 'state/posts/selectors';
+
+function PostTypeListPostThumbnail( { post } ) {
+	if ( ! post || ! post.canonical_image ) {
+		return null;
+	}
+
+	return (
+		<div className="post-type-list__post-thumbnail-wrapper">
+			<img
```

Comment:
>Would utilizing `ImagePreloader` be worthwhile here?  I suppose these are much smaller images, so they _should_ load faster

My only concern with this would be how it behaves with `<VirtualScroll />`, since that's a bit more render logic to be run every time a post is re-shown in the list.